WordPress/src/main/java/org/wordpress/android/ui/notifications/NotificationsDetailListFragment.java

```diff
@@ -304,6 +310,17 @@ private boolean hasNoteBlockAdapter() {
                                         mOnGravatarClickedListener
                                 );
                             }
+                        } else if (isFooterBlock(noteObject)) {
+                            noteBlock = new FooterNoteBlock(noteObject, mOnNoteBlockTextClickListener);
+                            ((FooterNoteBlock)noteBlock).setClickableSpan(
+                                    JSONUtils.queryJSON(noteObject, "ranges[last]", new JSONObject()),
+                                    mNote.getType()
+                            );
+
+                            if (mNote.isFollowType() || mNote.isReblogType() || mNote.isCommentType()) {
+                                // Request the reader post so that loading reader activities will work.
+                                ReaderPostActions.requestPost(mNote.getSiteId(), mNote.getPostId(), null);
```

Comment:
Does this need to request the post every time, or can it only be requested when `ReaderPostTable.postExists()` is false?